### ISSUE TITLE
fix the hacky java seed

### ISSUE THREAD

**Issue by miguelbiron** [Julia-Tempering/Pigeons.jl#28]
https://github.com/Julia-Tempering/Pigeons.jl/blob/46ebc06e514d74c74258bc6098f9fce68c4e861d/src/targets/StreamTarget.jl#L113-L116

hey @alexandrebouchard --- I was wondering if the following would be a less hacky alternative to the above

```julia
java_seed(rng::SplittableRandom) = (rand(split(rng), UInt64) >>> 1) % Int64
```

By doing the logical right shift, the sign bit is dropped, so the conversion always returns a positive Int64. Or am I missing something?

**Comment by alexandrebouchard** [comment]
Sounds great! :) I was in a hurry and that was not in the inner loop, but obviously an eye sore so I like your 1000 better. Feel free to create a PR with this. Thank you!!!

**Comment by miguelbiron** [comment]
closed by #35

**Comment by alexandrebouchard** [comment]
Thank you!
